docs/_articles/en/architecture/architecture.md

```diff
@@ -21,21 +21,35 @@ Version {{ site.data.versions.scanner }} of the Salesforce Code Analyzer plug-in
 
 ### Rule Catalog
 
-Each rule engine has a different set of rules and different formats for representing them. The Salesforce Code Analyzer plug-in communicates with each rule engine separately to pull the default rules they offer and populate them together in the rule catalog. This catalog contains the name of a rule, a short description, its classification category, and the code source language that the rule can analyze. Run the ```scanner:rule:list``` command to view this catalog. The output includes the rule engine name that a particular rule belongs to.
+Each rule engine has a different set of rules and different formats for representing them. Code Analyzer communicates with each rule engine separately to pull the default rules that they offer and to populate them together into a rule catalog. This catalog contains the name of a rule, a short description, its classification category, and the code source language that the rule can analyze.
 
-### Rule Engine Bridge
+Run the ```scanner:rule:list``` command to view the rule catalog. The command’s output includes the rule engine name that a particular rule belongs to.
 
-By unifying the representation of the rules into a rule catalog, Salesforce Code Analyzer can take a uniform set of input parameters. The bridge then detects the selected engines using the rules selected for input. It also determines the target files based on the file types defined in ```~/.sfdx-scanner/Config.json```. From the selected rules and target files, it tailors the input for each relevant rule engine and hands off the input to the rule engine for the actual scan.
+Example:
 
-After the scan completes and the rule engine provides the results, the bridge surfaces the results in a normalized format.
+|NAME								|LANGUAGES		|CATEGORIES				|RULESETS [DEP]								|ENGINE	  		|
+| ------- 							| ---------     | ---------				| ---------									| ---------		|
+|MyRule1							|visualforce	|Security				|Basic VF									|pmd			|
+|MyRule2							|apex			|design					|Complexity,Default ruleset...,quickstart 	|pmd			|
+|FormalParameterNamingConventions	|apex			|Code Style				|quickstart									|pmd			|
+|constructor-super					|javascript		|problem				|problem									|eslint			|
+|ApexFlsViolationRule				|apex			|Security				|											|sfge			|
 
-## Source Code and Contribution
+### Rule Engine Bridge
 
-The code for the Salesforce Code Analyzer plug-in is in [this GitHub repo](https://github.com/forcedotcom/sfdx-scanner). Salesforce is actively working on expanding and improving the tool. The project will soon be open for external contributions.
+By unifying the representation of the rules into a rule catalog, Code Analyzer accepts a uniform set of input parameters. Here’s how it works.
 
+1. Using rules that you select, the bridge detects the related engine and determines the target files based on the file types defined in ```~/.sfdx-scanner/{{ site.data.versions-v3.configfile }}```. 
```

Comment:
This one can be `Config.json` instead of the variable pattern, because it's 2.x.